File: spectra-serve-auth/src/main/java/com/yangxj96/spectra/server/auth/pojo/vo/AuthLogin.java
package com.yangxj96.spectra.server.auth.pojo.vo;

import jakarta.validation.constraints.NotBlank;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 登录请求参数
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class AuthLogin {

    /**
     * 用户名
     */
    @NotBlank(message = "用户名不能为空")
    private String username;

    /**
     * 密码
     */
    @NotBlank(message = "密码不能为空")
    private String password;
}

File: yangxj96-serve/yangxj96-serve-auth/src/main/java/com/yangxj96/saas/server/auth/pojo/vo/AuthLogin.kt
package com.yangxj96.saas.server.auth.pojo.vo

import jakarta.validation.constraints.NotBlank

/**
 * 登录请求参数
 */
class AuthLogin {

    /**
     * 用户名
     */
    @NotBlank(message = "用户名不能为空")
    var username: String? = null

    /**
     * 密码
     */
    @NotBlank(message = "密码不能为空")
    var password: String? = null
}
